buildSrc/src/main/kotlin/com/github/spotbugs/com.github.spotbugs.test.gradle.kts

```diff
@@ -39,7 +39,11 @@ testing {
                 all {
                     testTask.configure {
                         description = "Runs the functional tests."
-                        systemProperty("snom.test.functional.gradle", System.getProperty("snom.test.functional.gradle", gradle.gradleVersion))
+                        var testGradleVersion = providers.gradleProperty("snom.test.functional.gradle").getOrElse("current")
```

Comment:
A typo here, it should be `providers.systemProperty` or use Gradle property, fixing this in gradleVersion